projects/core/components/svg/svg.style.less

```diff
@@ -1,14 +1,12 @@
-@default-size: 1.5rem;
-
 :host {
     display: inline-flex;
     vertical-align: middle;
     flex-shrink: 0;
     align-items: center;
     justify-content: center;
     line-height: 0;
-    height: @default-size;
-    width: @default-size;
+    height: var(--tui-svg-size, 1.5rem);
```

Comment:
But that's on the host, what's the point of it?

Reply:
but if your icons

```html
<svg height="inherit" width="inherit" />
```

you can immediately set the overall size for all icons

<img width="368" alt="image" src="https://github.com/taiga-family/taiga-ui/assets/12021443/51a05164-91bb-4e84-a43f-941a27455c90">